testsuite/README.MD

```diff
@@ -90,6 +90,12 @@ the usage of `standalone-microprofile.xml` configuration file:
 
 > mvn clean verify -Dserver.home=PATH_TO_WIDLFLY_HOME -Dts.standalone.microprofile
 
+### Bootable jar testing
+Run testsuite or test against Bootable jar, not against Wildfly:
+
+> mvn clean verify -Dts.bootable -Ddefault=false -Ddisable.microprofile.tests -Dserver.version=21.0.0.Beta1-SNAPSHOT -Dserver.home=placeholder 
```

Comment:
What is `-Dserver.home` used for when testing Bootable JAR?

Reply:
This disable [download profile](https://github.com/resteasy/Resteasy/blob/master/testsuite/integration-tests/pom.xml#L58-L64), which is not necessary for bootable jar testing

Reply:
Makes sense, thanks, although I'd change it to something like `not-needed-for-bootable.jar`, otherwise a user could not understand at first glance. WDYT?

Reply:
Sure, updated

Reply:
Marking this thread as resolved, let me know if you have some objections ...